src/UpgradeableProxied.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
Set license to `// SPDX-License-Identifier: AGPL-3.0-or-later` in all files and add the license file in root.

Reply:
Updated